tests/ossfuzz/fuzz_common.h

```diff
@@ -0,0 +1,43 @@
+#include <inttypes.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include "lzma.h"
+
+
+// Output buffer for decompressed data. This is write only; nothing cares
+// about the actual data written here.
+static uint8_t OUTBUF[4096];
+
+// Some header values can make liblzma allocate a lot of RAM
+// (up to about 4 GiB with liblzma 5.2.x). We set a limit here to
+// prevent extreme allocations when fuzzing.
+const uint64_t MEM_LIMIT = 300 << 20; // 300 MiB
+
+
+void fuzz_code(lzma_stream *stream, const uint8_t *inbuf, size_t inbuf_size) {  
+    // Give the whole input buffer at once to liblzma.
+	// Output buffer isn't initialized as liblzma only writes to it.
```

Comment:
It looks like there is a mix of tabs and spaces. In .c files we use tabs, and treat the tab width as 8 characters. I'm sure other projects do this differently since there is no right answer to the tabs vs spaces debate